src/scribe-plugin-link-prompt-command.js

```diff
@@ -30,6 +31,11 @@ define(function () {
           link = passedLink;
         }
 
+        if(!checks.emptyLink(link)) {
```

Comment:
Alternatively, would trimming the link make it falsy on line 57 below, hence reusing existing logic, while also preventing broken links due to leading/trailing space?